infrastructure/ecs-main/serverless.yml

```diff
@@ -156,6 +156,19 @@ custom:
     prd: "prd"
     dotcomstg: "preprd"
     dotcomprd: "prd"
+  site:
+    host:
+      dev: "docs-dev.mongodb.com" # NOTE: cannot be empty in redirect
```

Comment:
Is this intended to be a future dev environment?

Reply:
So this is intentionally not valid url currently. If we setup a future domain for dev, then I think this one would be a good fit. I could have used the actual s3 website url domain here, but that would include our aws account id.